sktime/distances/extra_things.py

```diff
@@ -0,0 +1,29 @@
+#if __name__ == "__main__":
```

Comment:
please rename file or remove if not necessary

Reply:
done